### ISSUE TITLE
Ignite v3 Broken - Missing build folder

### ISSUE THREAD

**Issue by ruddell** [infinitered/ignite#1412]
### What's going on?
The recent v3 release is broken, no commands work

### Steps to reproduce
```
npm install ignite-cli@latest
ignite
```
Results in error:
```
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module '/Users/ruddell/.nvm/versions/node/v10.15.2/lib/node_modules/ignite-cli/bin/../build/cli/check.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/ruddell/.nvm/versions/node/v10.15.2/lib/node_modules/ignite-cli/bin/ignite:28:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```
You can see there is no build folder in the v3.0.0 release by doing the following:
```
mkdir repro && cd repro
npm install ignite-cli@latest
ls node_modules/ignite-cli/ 
```
### `ignite doctor` results:
Not possible

**Comment by jamonholmgren** [comment]
Apologies, you caught us at a broken build (while releasing 3.0). We rolled back and are releasing a new version soon. cc @ryanlntn

**Comment by jamonholmgren** [comment]
It looks like 3.0.1 has been released. Can you check it, please?

**Comment by ryanlntn** [comment]
@jamonholmgren Looks like the issue is persisting on 3.0.1:

```
~/Code ❯❯❯ ignite new AndrossApp
internal/modules/cjs/loader.js:596
    throw err;
    ^

Error: Cannot find module '/Users/ryanlntn/.config/yarn/global/node_modules/ignite-cli/bin/../build/cli/check.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/ryanlntn/.config/yarn/global/node_modules/ignite-cli/bin/ignite:28:1)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
~/Code ❯❯❯ ignite --version                                                                                                                                                                               ⏎
3.0.1
```

**Comment by jamonholmgren** [comment]
Fixed in 3.0.3. Thanks!
